Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -197,6 +193,14 @@ The password must meet at least three out of the following four complexity rules
 3. at least one digit (0-9)
 4. at least one special character (punctuation)
 
+**Alternative**
```

Comment:
I am not sure "Alternative" is a good title for this section. I think a title like "Dictionary check" would be better.